tickets.php

```diff
@@ -85,6 +85,8 @@
                     'message' => $_POST['message']
                     );
             $vars['cannedattachments'] = $attachments->getClean();
+            $vars['lockCode'] = $_POST['lock_code'] ?: null;
+            $vars['form_token'] = $_POST['form_token'] ?: null;
```

Comment:
This is a result of doing validation in the wrong place... results is mudding `$vars`